core-bundle/src/Messenger/WebWorker.php

```diff
@@ -115,11 +115,21 @@ private function processTransport(string $transportName): void
 
         $this->webWorkerRunning = true;
 
-        $input = new ArrayInput([
+        $inputParameters = [
             'receivers' => [$transportName],
             '--time-limit' => 30,
             '--sleep' => 0,
-        ]);
+            '--no-reset' => true, // Do not reset the container between every message (otherwise web profiler profiles etc. would all be gone)
```

Comment:
I’d prefer to keep the reset and instead move the prio of the terminate listener below `-1024`